packages/components/src/components/contract-card/contract-card-items/sass/contract-card-dialog.scss

```diff
@@ -23,6 +23,39 @@
         .dc-input-wrapper__input {
             border: 1px solid var(--border-normal);
         }
+        &--currency {
+            position: absolute;
+            height: 3.2rem;
+            right: 4rem;
+            align-items: center;
+            justify-content: center;
+            display: flex;
+            background: transparent;
+            border-color: transparent;
+            z-index: 2;
+            color: inherit;
+
+            &-symbol {
+                font-size: 1.4rem;
+                line-height: 1.5;
```

Comment:
How about `Text` component?

Reply:
Ideally, yes. But we'll have to refactor `Input` component for that and that's very much out of scope here 😅